src/main/java/uk/gov/pay/adminusers/service/InviteFinder.java

```diff
@@ -32,6 +33,21 @@ public InviteFinder(InviteDao inviteDao, UserDao userDao) {
                         return Optional.of(invite);
                     }).orElse(Optional.of(invite));
                 })
-                .orElseGet(() -> Optional.empty());
+                .orElseGet(Optional::empty);
+    }
+
+
+    public List<Invite> findAll(String serviceId) {
+        return inviteDao.findAllByServiceId(serviceId)
+                .stream()
+                .filter(inviteEntity -> !(inviteEntity.isExpired() || inviteEntity.isDisabled()))
```

Comment:
This might be a little clearer as:

```java
.filter(inviteEntity -> !inviteEntity.isExpired())
.filter(inviteEntity -> !inviteEntity.isDisabled())
```

Reply:
👍 might read better but we go through the list twice instead of once, not super sure it's worth it for such a small condition. What do you think?